Sources/Request/Request+URLRequest.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Request+URLRequest.swift
+//  Cara
+//
+//  Created by Jelle Vandebeeck on 02/12/2018.
+//
+
+import Foundation
+
+extension Request {
+    func makeURLRequest(with configuration: Configuration) throws -> URLRequest {
+        let url = try makeURL(with: configuration)
+        var urlRequest = URLRequest(url: url)
+        urlRequest.httpMethod = method.httpMethod
+        urlRequest.allHTTPHeaderFields = makeHeaders(with: configuration)
+        urlRequest.httpBody = try makeBody()
+        return urlRequest
+    }
+    
+    private func makeURL(with configuration: Configuration) throws -> URL {
```

Comment:
ik had dit wss in een extensie op `Configuration` gezet, maar _potato potato_ 🥔🥔

Reply:
Maar dan moet je `Request` meegeven, dus komt mss wel op hetzelfde neer, niet?